tests/rgen/Microsoft.Macios.Generator.Tests/Classes/Data/Trampolines/ExpectedAVFoundationTrampolinesTrampolines.cs

```diff
@@ -0,0 +1,1369 @@
+			}
+			return default;
+		}
+
+		internal static unsafe global::ObjCRuntime.BlockLiteral CreateNullableBlock (global::AVFoundation.AVAudioEngineManualRenderingBlock? callback)
+		{
+			if (callback is null)
+				return default (global::ObjCRuntime.BlockLiteral);
+			return CreateBlock (callback);
+		}
+
+		[BindingImpl (BindingImplOptions.GeneratedCode | BindingImplOptions.Optimizable)]
+		internal static unsafe global::ObjCRuntime.BlockLiteral CreateBlock (global::AVFoundation.AVAudioEngineManualRenderingBlock callback)
+		{
+			delegate* unmanaged<global::System.IntPtr, uint, global::ObjCRuntime.NativeHandle, int*, global::System.IntPtr> trampoline = &Invoke;
+			return new global::ObjCRuntime.BlockLiteral (trampoline, callback, typeof (SDAVAudioEngineManualRenderingBlock), nameof (Invoke));
+		}
+	}
+	// TODO: generate trampoline for AVFoundation.AVAudioEngineManualRenderingBlock
+
+	[UnmanagedFunctionPointerAttribute (CallingConvention.Cdecl)]
+	[UserDelegateType (typeof (global::AVFoundation.AVAudioIONodeInputBlock))]
+	unsafe internal delegate global::ObjCRuntime.NativeHandle DAVAudioIONodeInputBlock (global::System.IntPtr block_ptr, uint frameCount);
+
+	/// <summary>This class bridges native block invocations that call into C#</summary>
+	static internal class SDAVAudioIONodeInputBlock
+	{
+		[Preserve (Conditional = true)]
+		[UnmanagedCallersOnly]
+		[UserDelegateType (typeof (global::AVFoundation.AVAudioIONodeInputBlock))]
+		internal static unsafe global::ObjCRuntime.NativeHandle Invoke (global::System.IntPtr block_ptr, uint frameCount)
+		{
+			var del = global::ObjCRuntime.BlockLiteral.GetTarget<global::AVFoundation.AVAudioIONodeInputBlock> (block_ptr);
+			if (del is not null)
+			{
+				var ret = del (frameCount);
+				return global::ObjCRuntime.Runtime.RetainAndAutoreleaseNativeObject (ret);
+			}
+			return default;
+		}
+
+		internal static unsafe global::ObjCRuntime.BlockLiteral CreateNullableBlock (global::AVFoundation.AVAudioIONodeInputBlock? callback)
+		{
+			if (callback is null)
+				return default (global::ObjCRuntime.BlockLiteral);
+			return CreateBlock (callback);
+		}
+
+		[BindingImpl (BindingImplOptions.GeneratedCode | BindingImplOptions.Optimizable)]
+		internal static unsafe global::ObjCRuntime.BlockLiteral CreateBlock (global::AVFoundation.AVAudioIONodeInputBlock callback)
+		{
+			delegate* unmanaged<global::System.IntPtr, uint, global::ObjCRuntime.NativeHandle> trampoline = &Invoke;
+			return new global::ObjCRuntime.BlockLiteral (trampoline, callback, typeof (SDAVAudioIONodeInputBlock), nameof (Invoke));
+		}
+	}
+	// TODO: generate trampoline for AVFoundation.AVAudioIONodeInputBlock
+
+	[UnmanagedFunctionPointerAttribute (CallingConvention.Cdecl)]
+	[UserDelegateType (typeof (global::AVFoundation.AVAudioInputNodeMutedSpeechEventListener))]
+	unsafe internal delegate void DAVAudioInputNodeMutedSpeechEventListener (global::System.IntPtr block_ptr, global::System.IntPtr event);
```

Comment:
Bug, should be @event and not event. There seems to be an issue on how we are getting the parameter name from the delegate declaration:
```csharp
delegate void AVAudioInputNodeMutedSpeechEventListener (AVAudioVoiceProcessingSpeechActivityEvent @event);
```
For some reason we are losing the '@' which should not happen.

Reply:
Fix in https://github.com/dotnet/macios/pull/22930